resources/analysisTools/snvPipeline/snvCalling.sh

```diff
@@ -66,8 +71,23 @@ filenameMPileupError="${MPILEUP_SUBDIR}/${MPILEUPOUT_PREFIX}${PID}.${chr}.error"
 
 FILENAME_VCF_SNVS_TEMP=${FILENAME_VCF_SNVS}.tmp
 
-#[[ ! -f $filenameMPileupTemp ]] && 
-${BCFTOOLS_BINARY} mpileup ${MPILEUP_OPTS} -r ${chr} -f ${REFERENCE_GENOME} $tumorbamfullpath | ${BCFTOOLS_BINARY} call ${BCFTOOLS_OPTS} > $filenameMPileupTemp
+## For variant calling from ALT and HLA region in the GRCh38 genome
+# The *_CONTIGS_FILE contains the list of the contigs names
+MPILEUP_REGION_OPTION=""
+if [[ "$chr" == "HLA" ]]
```

Comment:
I don't understand this. The HLA contigs all have different names with an `HLA-` prefix. Similar for the ALT contigs, that all identify by an `_alt` suffix.

But where is a string-identity check? So this script is probably called for all the HLA and ALT contigs, but where are the different HLA- and _alt contigs mapped to the "ALT" and "HLA" strings? The CHROMOSOME_INDICES also only contain the "ALT" and "HLA" strings.

Reply:
The HLA and ALT contig names are provided in a text files `CHROMOSOME_HLA_CONTIGS_FILE` and `CHROMOSOME_ALT_CONTIGS_FILE`. And the calling options are updated accrodingly whether its for ALT/HLA or primary chrs.